extension/src/services/binding.ts

```diff
@@ -235,7 +237,26 @@ export default class Binding {
                 };
                 this.subtitleController.notification('info.enabledFastForwardPlayback');
                 break;
+            case PlayMode.repeat:
+                if (!this.repeatIntervalId) {
```

Comment:
Instead of scheduling and saving a new interval I would suggest just subscribing to the existing onWillStopShowing event instead. This would reduce the amount of additional state you need to manage, as currently you are having to schedule and unschedule the interval.